apscheduler/executors/asyncio.py

```diff
@@ -1,8 +1,8 @@
-from asyncio import iscoroutinefunction
 import sys
 
 from apscheduler.executors.base import BaseExecutor, run_job, run_coroutine_job
 
+from apscheduler.util import iscoroutinefunction_partial
```

Comment:
This belongs in the same import group as the previous import.

Reply:
I will ensure these abut.